packages/frontend/amp/components/lib/Elements.tsx

```diff
@@ -1,11 +1,12 @@
-import { TextBlockComponent } from '@frontend/amp/components/elements/TextBlockComponent';
-
 import React from 'react';
+
+import { TextBlockComponent } from '@frontend/amp/components/elements/TextBlockComponent';
 import { ImageBlockComponent } from '@frontend/amp/components/elements/ImageBlockComponent';
 import { InstagramBlockComponent } from '@frontend/amp/components/elements/InstagramBlockComponent';
 import { TweetBlockComponent } from '@frontend/amp/components/elements/TweetBlockComponent';
 import { CommentBlockComponent } from '@frontend/amp/components/elements/CommentBlockComponent';
 import { RichLinkBlockComponent } from '@frontend/amp/components/elements/RichLinkBlockComponent';
+import { SoundcloudBlockComponent } from '../elements/SoundcloudBlockComponent';
```

Comment:
How about @frontend/... instead of ../.. ?

Reply:
Auto import sorry